examples/okta_user/all_attributes.tf

```diff
@@ -31,4 +31,5 @@ resource "okta_user" "test" {
   title              = "Director"
   user_type          = "Employee"
   zip_code           = "11111"
+  password           = "Abcd1234"
```

Comment:
Had to add the `password` variable to all setup files in `examples/okta_user` directory, as tests would fail.
This is because in the update test, the user should have a previous password to update the current password. If not, okta api returns an error. I don't know if you prefer this, but another way is to probably have a separate method/test to check for password update. I thought this was easier, not sure if cleaner though.

Reply:
Can we keep this test case separate? Just create another password specific test case? That way we can keep all of these existing test cases as is. I want to be sure all of the old tests pass as is (no passwords changed or set) while also testing the new functionality.